app/src/main/java/net/bible/android/view/activity/StartupActivity.kt

```diff
@@ -312,7 +313,7 @@ open class StartupActivity : CustomTitlebarActivityBase() {
     }
 
     private suspend fun gotoCalculator() {
-        val show = CommonUtils.settings.getBoolean("show_calculator", false)
+        val show = BuildVariant.Appearance.isDiscrete || CommonUtils.settings.getBoolean("show_calculator", false)
```

Comment:
This does not make sense to me because calculator need to be set up first and is completely optional feature for discrete user too. Shouldn't it be?

Reply:
I would say that the calculator is not optional for the discrete variant, and that choosing to install the discrete variant APK is the user understanding that they do want the app to hide itself on startup.
Certainly the discrete variant is at odds with the UX goal of discoverability, and certainly the users need a way to learn what they can do with the app.  I would suggest that word-of-mouth may be one of the best ways for this to be shared, and it can be documented where it is distributed (although we don't want that to become too well-known to those who are adversarial).